pkg/scheduler/scheduler.go

```diff
@@ -334,6 +334,12 @@ func (s *Scheduler) doScheduleBinding(namespace, name string) (err error) {
 		}
 		return err
 	}
+	if !rb.DeletionTimestamp.IsZero() {
+		s.recordScheduleResultEventForResourceBinding(rb, nil, fmt.Errorf("skip schedule deleting resourceBinding: %s/%s", rb.Namespace, rb.Name))
```

Comment:
Should we emit an event for this scenario?
I suppose, no one would expect the ResourceBing would be scheduled during deletion phase, right?
So, why do they need a warning event?

Sorry, I didn't realize it during last review.

Reply:
maybe the administrator didn't expect that his binding was accidentally deleted by others, so I think it is more reasonable to emit an event.
in addition, referring to the design of k8s in this scenario.
https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/schedule_one.go#L370

Reply:
Thanks, I'm sold.